apps/frontpage/lib/get-tree.ts

```diff
@@ -27,13 +27,15 @@ export const generateDocsTree = (pathToFiles?: string, docsRoot?: string) => {
   const newPath = pathToFiles || 'content/docs';
   const newDocsRoot = docsRoot || newPath;
 
-  const files = fs.readdirSync(newPath);
```

Comment:
This path could not be read when deployed without this change.

See more: https://vercel.com/guides/how-can-i-use-files-in-serverless-functions#using-process.cwd()

Reply:
Damn. Still doesn't work. [Logs](https://vercel.com/storybookjs/web/logs?slug=app-future&slug=en-US&slug=storybookjs&slug=web&slug=logs&page=1&timeline=past30Minutes&startDate=1718738690922&endDate=1718740490922&selectedLogId=171874047458047458096300000&selectedLogTimestamp=1718740474580&forceEndDate=1718740476165)